packages/uiweb/src/lib/dataProviders/SpaceDataProvider.tsx

```diff
@@ -1,17 +1,34 @@
-import { useState } from "react";
+import { useState, createContext } from "react";
 import { SpacesUI } from "../components";
-import { SpaceDataContext } from "../context";
+
 import { ISpaceDataContextValues } from "../context/spacesContext";
+import { ThemeProvider } from "../components/space/theme/ThemeProvider";
+import { Theme } from "../components/space/theme";
 
-export interface ISpacesUIProviderProps {
+export interface ISpacesUIProviderProps  {
   spaceUI: SpacesUI;
+  customTheme: Theme;
```

Comment:
instead of customTheme as param, let's name it as 'theme' so as to make it generic